### PR TITLE
feat: validators testing script

### PR DESCRIPTION
**Pull request by ratik**

### CODE REVIEW

File: quark/testcases/ica/README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+    
+    <aside>
+    💡 Store the address of the contract somewhere. If will be needed for IBC relayer to whitelist (allow) operations only from this contract.
+    
+    </aside>
+    
+
+## Getting ready to test on CosmosHub
+
+You don't have `atom` you can go through the following steps:
```
Comment (foxpy) [inline quark/testcases/ica/README.md:22]:
```suggestion
If you don't have `atom`, then you can go through the following steps:
```

File: quark/testcases/ica/README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+<aside>
+💡 Write newly created neutron connection id’s somewhere - they are needed for running scripts
+
+</aside>
+
+## Test
+
+- Upload [testing script](https://github.com/neutron-org/neutron-contracts/blob/fae4fbfb6496fb9eeea6e7a3b39dfb0816a4a6fe/validator_test.sh) to your node
+- run this script on your server `./validator_test.sh /place/where/contracts/were/uploaded/to/neutron_validators_test.wasm YOUR_CONNECTION_ID`
```
Comment (foxpy) [inline quark/testcases/ica/README.md:62]: People usually are not that verbose when specifying path prefixes:
```suggestion
- run this script on your server `./validator_test.sh /path/to/contracts/neutron_validators_test.wasm YOUR_CONNECTION_ID`
```

**Review by foxpy — CHANGES_REQUESTED** [review]

**Review by foxpy — COMMENTED** [review]
Looks good, yet I am going to test it further today.

File: quark/testcases/ica/README.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+You don't have `junox` you can go through the following steps:
+
+- if you don't have Keplr [install it](https://www.keplr.app/)
+- if you don't have uni (Juno's testnet) in your Keplr go to this [Jsfiddle](https://jsfiddle.net/superatik/L6bys84z/1/)
+- copy your Uni address
+- go to [Faucet](https://faucet.roguenet.io/) and get your `junox`
+
+## Prepare IBC relayer (Hermes)
+
+[Install Hermes for quark-1](https://www.notion.so/Install-Hermes-for-quark-1-8137dc949630488fb49a562660c745bb)
```
Comment (oopcode) [inline quark/testcases/ica/README.md:40]: please reference the instruction in our github https://github.com/neutron-org/testnets/blob/main/quark/ibc-relayer/instruction.md

**Review by foxpy — APPROVED** [review]

File: quark/testcases/ica/README.md (hunk 3 above)
Comment (foxpy) [reply quark/testcases/ica/README.md:40]: In fact, this instruction is not even public.

**Review by foxpy — CHANGES_REQUESTED** [review]
I will approve it back when links to private documentation are fixed.

File: quark/testcases/ica/README.md [hunk 4]
```diff
@@ -1 +1,64 @@
...
+```
+
+<aside>
+💡 Write newly created neutron connection id’s somewhere - they are needed for running scripts
+
+</aside>
+
+## Test
+
+- Upload [testing script](https://github.com/neutron-org/neutron-contracts/blob/fae4fbfb6496fb9eeea6e7a3b39dfb0816a4a6fe/validator_test.sh) to your node
```
Comment (foxpy) [inline quark/testcases/ica/README.md:61]: I don't like having a link to a particular commit hash here. Maybe just specify audit fixes branch and path to script?

Comment (foxpy) [reply quark/testcases/ica/README.md:61]: And why did you upload your script here by the way if you ask users to download it from another repo?

**Review by foxpy — CHANGES_REQUESTED** [review]

File: quark/testcases/ica/README.md (hunk 4 above)
Comment (ratik) [reply quark/testcases/ica/README.md]: fixed

File: quark/testcases/ica/README.md [hunk 5]
```diff
@@ -1 +1,64 @@
+# Validators Instruction to test testnet
 
+# Testing Quark testnet
+
+## Upload testing script
+- Upload the [script](./validator_test.sh) to validator machine 
+
+## Upload contracts
+
+- At first get artifacts from [https://github.com/neutron-org/neutron-contracts/tree/main/artifacts](https://github.com/neutron-org/neutron-contracts/tree/main/artifacts) (TODO: update when we get it merged)
```
Comment (foxpy) [inline quark/testcases/ica/README.md:10]: The branch sure is wrong, and I guess this TODO can be removed now.

**Review by foxpy — CHANGES_REQUESTED** [review]

**Review by foxpy — APPROVED** [review]

**Pull request merged**